src/protocol/parsers/parser.h

```diff
@@ -40,9 +40,9 @@ class Parser : public QObject
      * @brief Parse a single byte
      *
      * @param byte
-     * @return uint8_t
+     * @return bool if a valid message is available
```

Comment:
What is the reason for this?

Reply:
There is no need to check use the state to check for new messages. A simple bool removes the necessity to use parser states to detect a new message and remove unnecessary logic.